sympy/stats/tests/test_rv.py

```diff
@@ -181,7 +179,8 @@ def test_dependent_finite():
 
 def test_normality():
     X, Y = Normal('X', 0, 1), Normal('Y', 0, 1)
-    x, z = symbols('x, z', real=True, finite=True)
+    x = Symbol('x', real=True, finite=True)
+    z = Symbol('z', real=True, finite=True)
```

Comment:
Was the former problematic?

Reply:
I was able to remove to remove the import of `symbols` as it was used here only.